docs/template_plugin/tests/functional/op_reference/base_reference_test.cpp

```diff
@@ -28,22 +28,21 @@ void CommonReferenceTest::Exec() {
 }
 
 void CommonReferenceTest::LoadNetwork() {
+    function->validate_nodes_and_infer_types();
```

Comment:
Why do we need to call `validate_and_infer` manually here?

CC @itikhono

Reply:
fixed